arch/arm/boot/dts/overlays/i2c-mlx90640-overlay.dts

```diff
@@ -0,0 +1,22 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2835";
+
+	fragment@0 {
+		target = <&i2c_arm>;
+		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+                        clock-frequency = <400000>;
```

Comment:
This line has ended up using spaces for indentation rather than TABs. Consistency is preferred.